src/main/ui/app/route/SearchPage.jsx

```diff
@@ -33,10 +33,8 @@ export default class SearchPage extends Component {
     doSearch() {
         const query = this.props.location.query.q;
         const type = this.props.location.query.type;
-        const searchKey = `type:${type ? type : "all"} query:${query ? query : ""}`;
-
-        console.log(searchKey);
-        console.log(this.state.latestSearch);
+	const tag = this.props.location.query.tag;
```

Comment:
Pls Note: This looks out of order for some reason but on viewing the file, it is perfectly indented